src/main/java/com/qaprosoft/carina/demo/gui/pages/OpinionsPage.java

```diff
@@ -0,0 +1,79 @@
+package com.qaprosoft.carina.demo.gui.pages;
+
+import com.qaprosoft.carina.core.foundation.webdriver.decorator.ExtendedWebElement;
+import com.qaprosoft.carina.core.gui.AbstractPage;
+import org.apache.log4j.Logger;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.util.List;
+
+public class OpinionsPage extends AbstractPage {
+    private Logger LOGGER = Logger.getLogger(OpinionsPage.class);
+
+    @FindBy(xpath = "//h1[@class='specs-phone-name-title']")
+    private ExtendedWebElement headText;
+
+    @FindBy(id = "sort-comments")
+    private ExtendedWebElement sortBy;
+
+    @FindBy(xpath = "//option[@value='2']")
+    private ExtendedWebElement bestRating;
+
+    @FindBy(xpath = "//span[@class='thumbs-score']")
+    private List<ExtendedWebElement> scoreCommentList;
+
+    @FindBy(xpath = "//a[@class='voting-link vote-up']")
+    private List<ExtendedWebElement> goodRateCommentList;
```

Comment:
I think it's better to create separate element for comment(opinion).This component has author, date, rating,This component also has some methods like upwote/reply...How do You think?